workflows/wf_titan_illumina_se.wdl

```diff
@@ -95,10 +95,10 @@ workflow titan_illumina_se {
     String  seq_platform = seq_method
 
     File    read1_clean                 = read_QC_trim.read1_clean
-    Int     fastqc_raw                  = read_QC_trim.fastqc_number_reads
-    String  fastqc_version              = read_QC_trim.fastqc_version
+    Int     fastq_scan_raw                  = read_QC_trim.fastq_scan_number_reads
```

Comment:
Should this be `fastq_scan_raw_number_reads`?

Reply:
Staying in-line with how we reported fastqc results, i.e. fastqc_raw & fastqc_clean